dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -946,93 +1012,107 @@ public List<HPCCRecord> readFile(HPCCFile file, Integer connectTimeoutMillis, bo
 
         return records;
     }
-
+    
     private void writeFile(List<HPCCRecord> records, String fileName, FieldDef recordDef, Integer connectTimeoutMs)
+    {
+        writeFile(records, fileName, recordDef, connectTimeoutMs, false);
+    }
+    
+    private void writeFile(List<HPCCRecord> records, String fileName, FieldDef recordDef, Integer connectTimeoutMs, boolean compressed)
     {
         try
         {
-            //------------------------------------------------------------------------------
-            //  Request a temp file be created in HPCC to write to
-            //------------------------------------------------------------------------------
+            writeFileWithExceptions(records, fileName, recordDef, connectTimeoutMs, compressed);
```

Comment:
is this test expecting to cause an exception, if so, the catch should fail, right?

Reply:
The emptyCompressedFileTest test is expecting an exception which is why I split that out into two functions. In that test I return if the exception is thrown so that the test succeeds and Assert.fail otherwise

Reply:
Ok, I think the writeFileWithExceptions name is causing confusion. At face-value that function would seem to write a file which will cause exceptions, but it seems to write a file, and report Exceptions if encountered. A slight name adjustment to something like writeFileAndReportAnyExceptions would avoid confusion.

Reply:
Yeah, that is a better name. I will make that change.